package.json

```diff
@@ -2,16 +2,25 @@
   "name": "node.js_course_practice",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/index.ts",
   "scripts": {
-    "start:dev": "nodemon src/index.js"
+    "start:dev": "nodemon src/index.ts"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/express": "^4.17.18",
```

Comment:
This dependency should be in devDependencies section. As `nodemon` too. All dependenciess that used just for development should be moved to dev section.

Reply:
Also, I found that `ts-node` dependency is missing in dev dependencies too. Probably, it installed globally on you machine if it works without it in dev dection. Better to have all required dependencies in local project's package.json file.

```
npm i -D nodemon @types/express ts-node
```

Reply:
Done